tests/system/action/motion_category/test_update.py

```diff
@@ -0,0 +1,44 @@
+from openslides_backend.shared.exceptions import DatabaseException
+from tests.system.action.base import BaseActionTestCase
+from tests.util import get_fqid
+
+
+class MotionCategorySystemTest(BaseActionTestCase):
+    def test_update_correct(self) -> None:
+        self.create_model(get_fqid("meeting/222"), {"name": "name_xQyvfmsS"})
+        self.create_model(
+            get_fqid("motion_category/111"),
+            {"name": "name_srtgb123", "prefix": "prefix_JmDHFgvH", "meeting_id": 222},
+        )
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "motion_category.update",
+                    "data": [{"id": 111, "name": "name_Xcdfgee"}],
+                }
+            ],
+        )
+        self.assertEqual(response.status_code, 200)
+        self.assert_model_exists(get_fqid("motion_category/111"))
+        model = self.datastore.get(get_fqid("motion_category/111"))
+        assert model.get("name") == "name_Xcdfgee"
+
+    def test_update_wrong_id(self) -> None:
+        self.create_model(get_fqid("meeting/222"), {"name": "name_xQyvfmsS"})
+        self.create_model(
+            get_fqid("motion_category/111"),
+            {"name": "name_srtgb123", "prefix": "prefix_JmDHFgvH", "meeting_id": 222},
+        )
+        with self.assertRaises(DatabaseException):
+            self.client.post(
+                "/",
+                json=[
+                    {
+                        "action": "motion_category.update",
+                        "data": [{"id": 112, "name": "name_Xcdfgee"}],
+                    }
+                ],
+            )
+        model = self.datastore.get(get_fqid("motion_category/111"))
+        assert model.get("name") == "name_srtgb123"
```

Comment:
Same as for create: test that all possible properties can be updated

Reply:
Done.